### PR TITLE
Fix crash with keybase --standalone log send

### PR DESCRIPTION
**Pull request by akalin-keybase**
This makes it work again, albeit without UI logs.

### CODE REVIEW

**Comment by zapu** [comment]
Grep for `func (d *Service) StartStandaloneChat(g *libkb.GlobalContext) error` and `StartStandaloneChat` in general to see how I solved it for chat in standalone. Maybe logsends can be made functional in standalone mode as well in similar manner. Maybe just using calling StartStandaloneChat could work, however unintuitive it might be, because it sets up connection manager.

**Comment by akalin-keybase** [comment]
Oh, this makes log send with `--standalone` work. The connection manager is just to get the GUI logs, so the returned error is just printed out while the rest of the logs are uploaded.

**Comment by zapu** [comment]
oh, so it does work with this PR? I understood it only bails out in nicer way then crashing, but I just hlaned at the code.

File: go/chat/tlf.go [hunk 1]
```diff
@@ -34,7 +35,7 @@ func NewKBFSNameInfoSource(g *globals.Context) *KBFSNameInfoSource {
 
 func (t *KBFSNameInfoSource) tlfKeysClient() (*keybase1.TlfKeysClient, error) {
 	if t.G().ConnectionManager == nil {
-		return nil, fmt.Errorf("no connection manager available")
```
Comment (akalin-keybase) [inline go/chat/tlf.go]: tiny cleanup

File: go/service/git.go [hunk 2]
```diff
@@ -340,7 +341,7 @@ func (h *GitHandler) kbfsClient() (*keybase1.KBFSGitClient, error) {
 		return nil, libkb.LoginRequiredError{}
 	}
 	if h.G().ConnectionManager == nil {
-		return nil, fmt.Errorf("no connection manager available")
```
Comment (akalin-keybase) [inline go/service/git.go]: tiny cleanup

**Comment by akalin-keybase** [comment]
Yeap! Updated PR description to mention that.

**Comment by zapu** [comment]
Max is out till Monday BTW

**Comment by akalin-keybase** [comment]
Oh, yeah, thanks! Kicking over to @patrickxb

**Comment by akalin-keybase** [comment]
Oops, didn't see your approval @zapu. Going ahead and merging!

**Pull request merged**